Nos/Models/CoreData/Nos.xcdatamodeld/Nos 21.xcdatamodel/contents

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="23507" systemVersion="23G93" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="Author" representedClassName=".Author" syncable="YES">
+        <attribute name="about" optional="YES" attributeType="String"/>
+        <attribute name="displayName" optional="YES" attributeType="String"/>
+        <attribute name="hexadecimalPublicKey" attributeType="String"/>
+        <attribute name="lastUpdatedContactList" optional="YES" attributeType="Date" usesScalarValueType="NO"/>
+        <attribute name="lastUpdatedMetadata" optional="YES" attributeType="Date" usesScalarValueType="NO"/>
+        <attribute name="lastUpdatedMuteList" optional="YES" attributeType="Date" usesScalarValueType="NO"/>
+        <attribute name="muted" optional="YES" attributeType="Boolean" usesScalarValueType="YES"/>
+        <attribute name="name" optional="YES" attributeType="String"/>
+        <attribute name="nip05" optional="YES" attributeType="String"/>
+        <attribute name="profilePhotoURL" optional="YES" attributeType="URI"/>
+        <attribute name="pronouns" optional="YES" attributeType="String"/>
+        <attribute name="rawMetadata" optional="YES" attributeType="Binary"/>
+        <attribute name="rawMetadata1" optional="YES" attributeType="Binary"/>
+        <attribute name="website" optional="YES" attributeType="String"/>
+        <relationship name="events" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="Event" inverseName="author" inverseEntity="Event"/>
+        <relationship name="followers" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="Follow" inverseName="destination" inverseEntity="Follow"/>
+        <relationship name="followNotifications" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="NosNotification" inverseName="follower" inverseEntity="NosNotification"/>
+        <relationship name="follows" optional="YES" toMany="YES" deletionRule="Cascade" destinationEntity="Follow" inverseName="source" inverseEntity="Follow"/>
+        <relationship name="includedInLists" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="AuthorList" inverseName="authors" inverseEntity="AuthorList"/>
+        <relationship name="incomingNotifications" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="NosNotification" inverseName="user" inverseEntity="NosNotification"/>
+        <relationship name="lists" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="AuthorList"/>
```

Comment:
Xcode is unhappy that this "lists" relationship has no inverse. It seems the same as "includedInLists" though. Did you mean to have both of them?

<img width="1178" alt="Nos_—_Nos_21_xcdatamodel" src="https://github.com/user-attachments/assets/d27c88b1-c0ff-4776-ad1f-1f116cf15e1d" />

Reply:
But I see now that if I set the inverse of `lists` to `author`, then `events` no longer has the inverse set to `author`. So that's a problem.

Reply:
I went ahead and added another relationship to AuthorList: `owner`, which goes to `Author`, and allows us to set the `Author` inverse to `owner`.